cypress/integration/Tooltip.spec.ts

```diff
@@ -155,14 +155,14 @@ describe('Tooltip', () => {
     });
 
     it('the span element should not have an aria-describedby attribute', () => {
-      cy.get('button').should('not.have.attr', 'aria-describedby');
+      cy.get('span').should('not.have.attr', 'aria-describedby');
     });
 
     it('the span element should not have an aria-label attribute', () => {
-      cy.get('button').should('not.have.attr', 'aria-describedby');
+      cy.get('span').should('not.have.attr', 'aria-describedby');
     });
 
-    context('when the "Delete" button is hovered', () => {
+    context('when the "Some Text" text is hovered', () => {
```

Comment:
This was a copy/paste error. The tooltip isn't from a delete button, but a span with some text.